siriuspy/siriuspy/clientconfigdb/_templates.py

```diff
@@ -18,13 +18,13 @@
 _FLOAT_TYPES = {float}
 
 
-for k, tp in _np.typeDict.items():
-    if isinstance(k, str) and k.startswith('int'):
+for _k, tp in _np.typeDict.items():
+    if isinstance(_k, str) and _k.startswith('int'):
```

Comment:
I don't know... I think in cases like this it is better to use `key` instead of `_k`

Reply:
done.